Description:  Don't try and build cddlib.
 Upstream always builds it against built mpir, but we want to use the Debian
 package.
Author: Doug Torrance
Last-Update: 2017-06-21

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -891,10 +891,6 @@
 BUILD_ALWAYS="givaro $BUILD_ALWAYS"
 # givaro uses gmp, but we replace gmp with mpir (a patched version), so we must always build givaro
 
-BUILD_cddlib=yes
-BUILD_ALWAYS="cddlib $BUILD_ALWAYS"
-# cddlib uses gmp, but we replace gmp with mpir (a patched version), so we must always build cddlib
-
 if [[ $BUILD_mpir = yes ]]
 then AC_MSG_NOTICE([mpir is being built (to replace gmp), so we will build pari])
      BUILD_pari=yes
